### PR TITLE
Generating SDK from the TypeSpec commit

### PR DESCRIPTION
**Pull request by hamshavathimunibyraiah**
Generating SDK from the latest commit from TypeSpec PR : https://github.com/Azure/azure-rest-api-specs/pull/30416

### CODE REVIEW

File: sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_patch.py [hunk 1]
```diff
@@ -295,9 +295,7 @@ def begin_translation(
         """
 
     @overload
```
Comment (kristapratico) [inline sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_patch.py:297]: Given we renamed the internal method _begin_start_translation --> _begin_translation, we'll need to update where we call this method in the _patch on L471 and in the async file

Comment (hamshavathimunibyraiah) [reply sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_patch.py:297]: Updated

File: sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_operations/_operations.py [hunk 2]
```diff
@@ -52,7 +52,7 @@
 _SERIALIZER.client_side_validation = False
 
 
-def build_document_translation__begin_start_translation_request(  # pylint: disable=name-too-long
+def build_document_translation__begin_translation_request(  # pylint: disable=name-too-long
```
Comment (kristapratico) [inline sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_operations/_operations.py:55]: Sorry missed commenting that we need to update for the renames in the sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_operations/_patch.py + async as well

Comment (hamshavathimunibyraiah) [reply sdk/translation/azure-ai-translation-document/azure/ai/translation/document/_operations/_operations.py:55]: Updated

**Review by kristapratico — APPROVED** [review]

**Pull request merged**